src/kubeobjects/time.go

```diff
@@ -0,0 +1,11 @@
+package kubeobjects
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func IsOutdated(last, now *metav1.Time, threshold time.Duration) bool {
```

Comment:
```suggestion
func IsOutdated(previous *metav1.Time, current *metav1.Time, threshold time.Duration) bool {
```
I also don't necessarily like `threshold` but I can't think of a better name